src/main/java/com/github/bartimaeusnek/cropspp/crops/TConstruct/AluminiumOreBerryCrop.java

```diff
@@ -32,22 +32,9 @@ public ItemStack getGain(ICropTile crop) {
 
     @Override
     public int growthDuration(ICropTile crop) {
-        int r;
-        if (ConfigValues.debug)
-            r = 1;
-            // Same growth stages as melons and pumpkins
-        else if (crop.getSize() >= 2) {
-
-            // Ripens "quickly"
-            r = 3000;
-        } else if (crop.getSize() == 3 && crop.isBlockBelow("blockAluminium"))
-            r = 1500;
-        else {
-            // Takes a while to grow from seed
-            r = 500;
+        if (ConfigValues.debug) return 1;
+        return crop.getSize() >= 2 ? 3000 : 500;
```

Comment:
Same here. `crop.isBlockBelow("blockAluminium")` is missing (mistake or intended?)

Reply:
Intended, handled by canGrow()